CHANGELOG.md

```diff
@@ -2,6 +2,16 @@
 
 All notable changes to this project will be documented in this file, in reverse chronological order by release.
 
+## NEXT
+
+### Fixed
+- [#9](https://github.com/zendframework/zend-expressive-authentication-basic/pull/9)
+  If the decoded authentication string did contain a colon, a PHP Warning was thrown in PHP 7.2
+ 
+
+- [#9](https://github.com/zendframework/zend-expressive-authentication-basic/pull/9)
+  It was not possible to use passwords that contained colons
```

Comment:
Please use the same format as previous changelogs:

```markdown
## 0.3.1 - TBD

### Added

- Nothing.

### Changed

- Nothing.

### Deprecated

- Nothing.

### Removed

- Nothing

### Fixed

- ...
```

Notes:

- There is a line between a header and the first line of text.
- Only one empty line between list items.
- Assume a bugfix version unless you are proposing a new feature or a breaking change.